Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSReductionCore.py

```diff
@@ -170,17 +167,19 @@ def PyExec(self):
         monitor_workspace = self._move(state_serialized, monitor_workspace, component_as_string)
 
         # --------------------------------------------------------------------------------------------------------------
-        # 5. Apply masking (pixel masking and time masking)
-        # --------------------------------------------------------------------------------------------------------------
-        progress.report("Masking ...")
-        workspace = self._mask(state_serialized, workspace, component_as_string)
-
-        # --------------------------------------------------------------------------------------------------------------
-        # 6. Convert to Wavelength
+        # 5. Convert to Wavelength
+        #    Note (5) and (6) have swapped order to avoid EventWS mask rebinning issue
+        #    look to swap them back when/if this is fixed
         # --------------------------------------------------------------------------------------------------------------
         progress.report("Converting to wavelength ...")
         workspace = self._convert_to_wavelength(state_serialized, workspace)
 
+        # --------------------------------------------------------------------------------------------------------------
+        # 6. Apply masking (pixel masking and time masking)
+        # --------------------------------------------------------------------------------------------------------------q
+        progress.report("Masking ...")
+        workspace = self._mask(state_serialized, workspace, component_as_string)
```

Comment:
I think you may need to change this method slightly here as the workspace it is masking has now been converted into a counts against wavelength workspace rather than a counts against TOF, This means the Bin Masking limits will also need to be transformed from TOF to wavelength. 

It also seems that a lot of the SANS system tests are failing following this change which may be for this reason or for some other reason which will need to be investigated.